app/templates/Users/login.php

```diff
@@ -0,0 +1,14 @@
+<div class="users form">
+  <?= $this->Flash->render() ?>
+  <h3>Login</h3>
+  <?= $this->Form->create() ?>
+  <fieldset>
```

Comment:
I think the indentation become 2 space only instead of 4 space.

I tried to copy the same from [CakePHP tutorial](https://book.cakephp.org/4/en/tutorials-and-examples/cms/authentication.html#:~:text=Add%20the%20template%20logic%20for%20your%20login%20action%3A) and it's using 4 space.

```
<div class="users form">
    <?= $this->Flash->render() ?>
    <h3>Login</h3>
    <?= $this->Form->create() ?>
    <fieldset>
        <legend><?= __('Please enter your username and password') ?></legend>
        <?= $this->Form->control('email', ['required' => true]) ?>
        <?= $this->Form->control('password', ['required' => true]) ?>
    </fieldset>
    <?= $this->Form->submit(__('Login')); ?>
    <?= $this->Form->end() ?>

    <?= $this->Html->link("Add User", ['action' => 'add']) ?>
</div>
```

Please kindly share the cause space becomes 2 only in this PR. I'm not sure, is it because the setting (IntelliJ IDEA) at your side?

Additionally, please refer https://book.cakephp.org/4/en/contributing/cakephp-coding-conventions.html#indentation